download-maven-plugin/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<groupId>com.googlecode.maven-download-plugin</groupId>
 	<artifactId>download-maven-plugin</artifactId>
 	<packaging>maven-plugin</packaging>
-	<version>1.2.2-SNAPSHOT</version>
+	<version>1.2.3-SNAPSHOT</version>
```

Comment:
No need to change that. The 1.2.2 isn't release yet, it's our "work-in-progress version"